tests/functional/common.js

```diff
@@ -72,25 +72,27 @@ export function reportCommandErrors(obj: Object, msg: ?string) {
   throw error;
 }
 
-// execCommand helper
+// execWebExt helper
 
-export type RunCommandResult = {|
+export type RunWebExtResult = {|
   exitCode: number,
   stderr: string,
   stdout: string,
 |};
 
-export type RunningCommand = {|
-  execPath: string,
+export type RunningWebExt = {|
   argv: Array<string>,
-  waitForExit: Promise<RunCommandResult>,
+  waitForExit: Promise<RunWebExtResult>,
   spawnedProcess: ChildProcess,
 |};
 
-export function execCommand(
-  execPath: string, argv: Array<string>, spawnOptions: child_process$spawnOpts,
-): RunningCommand {
-  const spawnedProcess = spawn(execPath, argv, spawnOptions);
+export function execWebExt(
+  argv: Array<string>, spawnOptions: child_process$spawnOpts,
+): RunningWebExt {
+
+  const spawnedProcess = spawn(
+      process.execPath, [webExt, ...argv], spawnOptions);
```

Comment:
nit: all block continuations should only be indented by two spaces. Sorry, we don't have a lint rule for it because it's too complicated :clown_face: